frontend/src/components/petlistings/PetlistingsFilter/index.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from 'react';
+
+import Button from '../../ui/Button';
+import PetlistingsFilterItem from '../PetlistingsFilterItem';
+
+import styles from './PetlistingsFilter.module.css';
+
+import { availableFilters } from '../../../constants/filter';
+
+function PetlistingsFilter({ setFilters }) {
+  const [searchFilters, setSearchFilters] = useState(availableFilters);
+  const [resetFilter, setResetFilter] = useState(false)
+  const [isFilterMenuOpen, setIsFilterMenuOpen] = useState(window.innerWidth > 832 ? true : false);
+
+  const clearFilter = () => {
+    console.log('clear all filters')
```

Comment:
remember to remove console.logs

Reply:
Fixed in [85a1a86](https://github.com/lujorryn/PetPal-G0346/pull/24/commits/85a1a86f83af31181ba802c724ed58323bc5967c)